whylog/tests/tests_front/tests_whylog_factory.py

```diff
@@ -0,0 +1,27 @@
+import os
+import shutil
+from whylog.front.whylog_factory import whylog_factory
+from whylog.log_reader import LogReader
+from whylog.teacher import Teacher
+
+
+class MockClass():
+    def __call__():
+        pass
+
+
+def tests_whylog_factory():
+    try:
+        log_reader, teacher_generator = whylog_factory()
+        teacher = teacher_generator()
+
+        # check if returned object classes
+        assert isinstance(log_reader, LogReader)
+        assert isinstance(teacher, Teacher)
+    except Exception as exc:
+        raise exc
+    finally:
```

Comment:
Please, tell me what do we test in following section. I see only removing some directory. Why do we test it?